### DISCUSSION TITLE
Merging two texts and keeping the differences with marks

### DISCUSSION THREAD

**Discussion by Villard** [WinMerge/winmerge#1781] [Q&A]
hello
I'm just starting using WinMerge. 
My final objective is to use it in command-line and to merge two texts in one text and showing the differences 

If Text1 is :
> It's slow comparing words 
> and characters. Instead,  
> WinMerge compares lines.
> Believe it or else.

and Text2 is
> It's slow comparing words.
> WinMerge compares lines.
> Believe it or not.

I would like the get this kind of output, with marks [Text1] and [Text2] showing the differences
> It's slow comparing words 
> [Text1]and characters. Instead, [Text1]  
> WinMerge compares lines.
> Believe it or [Text1]else. [Text1][Text2]not. [Text2]

How can I do this ? Thanks for your help
Villard

**Comment by sdottaka** [comment]
Unfortunately, WinMerge cannot produce such output.

The wdiff command can produce output in a similar format.

https://www.di-mgt.com.au/wdiff-for-windows.html
~~~
wdiff.exe file1.txt file2.txt
    It's slow comparing [-words
    and characters. Instead,-] {+words.+}
    WinMerge compares lines.
    Believe it or [-else.-] {+not.+}
~~~

**Reply by Villard** [reply]
Thank you for your reply. I'll take a look and try
